core/src/scss/components/hero/_hero--caption.scss

```diff
@@ -0,0 +1,45 @@
+@charset "UTF-8";
+
+.su-hero--caption {
+
+  // Ensure no card shows up.
+  .su-hero__card {
+    display: none;
+  }
+
+  // The media element shall not be absolute in this option.
+  .su-hero__media {
+    @include grid-media('md') {
+      max-height: map-get($_su-hero-height, 'md');
+      position: relative;
+    }
+
+    @include grid-media('lg') {
+      max-height: map-get($_su-hero-height, 'lg');
+      position: relative;
+    }
+
+    @include grid-media('xl') {
+      max-height: map-get($_su-hero-height, 'xl');
+      position: relative;
+    }
+  }
+
+  .su-hero__content {
+    @include centered-column;
+    @include modular-spacing(padding-top, 0);
+    @include modular-spacing(padding-bottom, 2);
+    text-align: right;
+    display: block;
+
+    p {
+      @include credits;
```

Comment:
The caption currently in this PR is larger and darker than it should be, but the credits/caption font sizes and colors are being worked on in my media component PR, so this would fix itself later after that's merged in.

Reply:
I'm overriding the style for the `<p>` element only and it will be up to the user to supply the correct html and styles they want.

Reply:
Sure, that makes total sense. Just want to document here that the base `<p>` will look different since `@credit` is being worked on.